### DISCUSSION TITLE
New UI no longer has handy "compare" options

### DISCUSSION THREAD

**Discussion by arielallon** [community/community#6218] [Other Feature Feedback, Questions, & Ideas] [answered]
The old UI included a dropdown link to allow comparing a given release to any other arbitrary release. The new UI removed that.
I think I primarily used that to compare the most recent release to master when looking at what all changed while composing a new release. The "auto generate release notes" helps alleviate that need, but there was still benefit in a bigger comparison.

Yes of course I can manually construct the URL to compare tags, but it's nice to have it mostly done for me already. 
Also notable that the old UI options never actually included `master` (or `main`), just tags, so I did have to modify the URL a bit from a tag comparison. But it was still handy.

**Comment by MylesBorins** [comment, marked as answer]
This is an unintentional regression, I've scoped a feature with engineering to fix this

**Reply by arielallon** [reply]
Excellent, thanks so much!

**Reply by kir4h** [reply]
> This is an unintentional regression, I've scoped a feature with engineering to fix this

Thanks! I switched back to the old one just for this reason

**Reply by MylesBorins** [reply]
We have an open PR to fix this that will hopefully land monday

**Reply by MylesBorins** [reply]
The fix has landed PTAL

**Reply by arielallon** [reply]
Perfecto. Thanks.
